lua/solarized-osaka/theme.lua

```diff
@@ -313,9 +313,27 @@ function M.setup()
     ["@module"] = { link = "Include" },
 
     -- tsx
-    ["@tag.tsx"] = { fg = c.green500 },
     ["@constructor.tsx"] = { fg = c.blue500 },
     ["@tag.delimiter.tsx"] = { fg = c.orange500 },
+    ["@tag.tsx"] = { fg = c.yellow500 },
+    ["@keyword.return.tsx"] = { fg = util.darken(c.green500, 0.85), style = "italic" },
+    ["@keyword.tsx"] = { fg = util.darken(c.green500, 0.85), style = "italic" },
```

Comment:
`util.darken` is prohibited. it messes up the color consistency.

Reply:
It was a very slight change to create emphasis on the italicized words but removed and only applying the `italic` style now.

Reply:
Any other util provided would result in the same issue so removed it and left only the style attribute.